apps/public-docsite-v9/src/Concepts/Upgrade/FromV8/Components/RadioGroup.stories.mdx

```diff
@@ -0,0 +1,197 @@
+import { Meta } from '@storybook/addon-docs';
+
+<Meta title="Concepts/Upgrading/from v8/Components/ChoiceGroup to RadioGroup Upgrade" />
+
+# ChoiceGroup to RadioGroup Upgrade
+
+Fluent UI v8 provides the `ChoiceGroup` control for presenting a list of radio options.
+In Fluent UI v9 `ChoiceGroup` is replaced with `RadioGroup`.
+
+While there are several differences between these controls the primary change is that `RadioGroup` accepts
+its options as child `Radio` components while `ChoiceGroup` accepts options via its `options` prop.
+
+## Examples
+
+### Basic Migration
+
+Basic usage of `ChoiceGroup` looks like
```

Comment:
In the near future, I'd like to have the MDX show stories inline (v8, a shim if it exists, and v9).  That way the reader can see the component in action, and the code.